ftw/tabbedview/browser/batching.pt

```diff
@@ -62,6 +54,14 @@
            tal:content="batch/lastpage">3457</a>
     </span>
 
+    <span class="next"
```

Comment:
@jone, is this the correct position or should the button go after the dynamic batching?